<launch>
	<node pkg="coax_server" type="coax_server" name="coax_server" output="screen" args="/dev/ttyS0"/>
</launch>